lib/parsers/open-api.js

```diff
@@ -1,6 +1,48 @@
 const _ = require('lodash')
 const JoiJsonSchemaParser = require('./json-draft-04')
 
+function isKnownMetaKey(key) {
+  return key.startsWith('x-') || key === 'deprecated' || key === 'readOnly' || key === 'writeOnly'
+}
+
+function isKnownKey(key) {
```

Comment:
Tried to stick as close to previous `_.pick` implementation as possible using `Set` for more optimised `.has` now that this parser is using `_.pickBy`